docs/platforms/native/advanced-usage/signal-handling/index.mdx

```diff
@@ -0,0 +1,62 @@
+* The [glibc-docs](https://www.gnu.org/software/libc/manual/html_node/Signal-Handling.html) and [the man pages](https://man7.org/linux/man-pages/man7/signal.7.html) offer reference details for the signal vocabulary.
+* We recommend chapters 20-22 of Michael Kerrisk's [The Linux Programming Interface](https://man7.org/tlpi/) if you want a book-length treatment.
+ 
+
+## Signals of interest
+
+Signals can support many events, not all of which signify an error or terminate the application (like `SIGCONT`, `SIGUSR1`, or `SIGUSR2`).
+
+In the context of the Native SDK, we are only interested in signals that will result in the application's unexpected termination (crashes). These typically include:
+
+* `SIGSEGV`: the process tried to access an invalid virtual memory "segment".
+* `SIGBUS`: the process tried to access an invalid physical address or a VM page failed to "page-in".
+* `SIGABRT`: raised when `abort()` was called. This is the only signal handled on Windows (and only by the `crashpad` backend) because calling `abort()` bypasses Structured Exception Handling ([`SEH`](https://learn.microsoft.com/en-us/windows/win32/debug/structured-exception-handling)).
+* `SIGFPE`: an erroneous arithmetic operation that occurred in the CPU.
+* `SIGILL`: the process attempted to execute an illegal (often privileged) processor instruction.
+* `SIGTRAP`: typically used to inform debuggers when execution hits a breakpoint instruction (`int 3`, `brk`, etc.), which instrumentation tools and sanitizers sometimes also issue.
+
+If you use the `crashpad` backend, in addition to the above, on Linux, you will also get crash reports for:
+
+* `SIGSYS`: the kernel received a system call with an invalid argument.
+* `SIGEMT`: an emulator trap occurred.
+* `SIGXCPU`: the process exceeded a previously specified CPU [rlimit](https://pubs.opengroup.org/onlinepubs/9699919799/functions/getrlimit.html).
+* `SIGXFSZ`: similar to `SIGXCPU`, raised when a process grows a file beyond a maximum allowed size ([`RLIMIT_FSIZE`](https://man7.org/linux/man-pages/man2/getrlimit.2.html)). 
+
+If your application currently installs handlers for any of the above signals, that can result in a potential conflict. Expressly
+
+* refrain from registering a conflicting signal handler in your application.
+* if you have to, register this handler after `sentry_init()` and let it invoke the previously installed handler (ours) after it finished or you won't get a report.
+* if you or one of your dependencies register a handler for the above signals _before_ you call `sentry_init()`, we will reinstall the previous handler as soon as you call `sentry_close()`. Only the `inproc` backend invokes the previously installed handler at the end of its handler.
+* if you want to act on any crash and not consider specific signals, [register an `on_crash()` hook](https://docs.sentry.io/platforms/native/configuration/filtering/#using-on_crash).
```

Comment:
I think referencing "previous handler" gets confusing. Can we simplify the wording by saying "your handler" and "Sentry's handler"? That would make it much clearer which handler we're referencing.

Reply:
Yeah, that makes a lot of sense. Thanks. Fixed it here: https://github.com/getsentry/sentry-docs/pull/9787/commits/fb22952c2f2d4e20122bbcf3243703fc958eba00